### PR TITLE
[resoto][feat] Add vulnerability badge

### PR DESCRIPTION
**Pull request by aquamatthias**
# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/>
-<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1></p>
+<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1>
 
-<p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_banner.png" /></p>
+<p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_banner.png" alt="Banner"/></p>
```
Comment (TheCatLady) [inline README.md:4]: It's actually best practice to omit the alt tag for decorative images.

Comment (aquamatthias) [reply README.md:4]: My linter complains - nevertheless removed it.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
 <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/>
-<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1></p>
+<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1>
```
Comment (TheCatLady) [inline README.md:2]: Should close the `</p>` tag in the previous line. Also, perhaps update this heading to `Sync infrastructure data to a single place.`?

Comment (aquamatthias) [reply README.md:2]: The paragraph one line above is already closed. Do you want to nest the h1 inside the <p>.
I will adjust the heading - good point.

**Review by meln1k — DISMISSED** [review]
LGTM!

File: README.md (hunk 2 above)
Comment (TheCatLady) [reply README.md]: > The paragraph one line above is already closed. Do you want to nest the h1 inside the `<p>`?

No, the `h1` element should not be nested within the paragraph tag. However, there is no closing tag on the above line.

Comment (TheCatLady) [reply README.md]: This is what I mean:
```diff
- <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/>
+ <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/></p>
```

Comment (TheCatLady) [reply README.md]: @aquamatthias The `img` tag is self-closing. The `p` tag enclosing the image is not.

Comment (aquamatthias) [reply README.md]: ah - overlooked it. you are right.

**Review by TheCatLady — APPROVED** [review]

**Pull request merged**